firestore/query_multiple_inequality.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package firestore
+
+// [START firestore_query_filter_compound_multi_ineq]
+import (
+	"context"
+	"fmt"
+	"io"
+
+	"cloud.google.com/go/firestore"
+)
+
+func multipleInequalities(w io.Writer, projectID string) error {
+	ctx := context.Background()
+	client, err := firestore.NewClient(ctx, projectID)
+	if err != nil {
+		return fmt.Errorf("firestore.NewClient: %v", err)
```

Comment:
Actually, one more change, please: change the verb to "%w" to wrap the error. For samples testing, this helps preserve the error payload for CI logs.

Reply:
Added